contributor/04-mechanics-of-contributing.md

```diff
@@ -0,0 +1,160 @@
+# Draft comments
+
+## Video playbook outline
+
+What are the nuts and bolts of advice to follow when acting as an InnerSource Contributor?  This segment describes the hands-on things to do to make your contribution a success.  We'll give practical tips to leverage when preparing to work on a contribution, during development, and also in the pull request. 
+
+## Outline doc
+
+See [Outline Section](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/outline.md#mechanics-of-contributing)
+
+# Article
+
+Are you ready to start contributing to other teams projects/repos?
+Do you look forward to reducing your blockers not by management escalation but by collaboration? 
+This section gives practical advice and highlights gotchas to remember when making an InnerSource contribution. It enables you and the host team to have as pleasant an experience as possible, setting the foundation for more contributions and great collaboration.  
+
+This article is separated into the three steps you will likely experience
+
+- Soliciting your contribution opportunity and preparing to work on it
+- Actually creating the contribution
+- Polishing and wrapping the gift nicely and presenting it to the host team.
+
+If your contribution is larger, you'll possibly go through (some) these steps repeatedly as you iterate towards your common goal.
+It is very likely that as you do this everything will feel more and more naturally - maybe you'll even wonder why you were doing anything else before.
+
+## Preparing to work
+
+### Lead times
+
+One key difference is the turn-around time. 
+With every first time contribution you are coming to a new (host) team.
+As a result, you'll need to get to know their code-base and the technology used, but also their preferred development environment (think test framework, build system).
+Even in cases where this type of tooling is standardized each team will have developed some individual peculiarities.
+In addition to the technical side, you may be faced with differences in communication, think code-reviews.
+Even if you are coming back after a while the teams' ways and members might have changed.
+Take your time as you would to catch up with a friend you haven't seen in a while and whom you are visiting now.
+
+Give yourself enough lead time.
+Start early enough so that your work is available for you to leverage at the time you need it. 
+It's better to add more slack time initially - you'll get a feeling about the turn-around times once you work with the host team.
+Often, you will notice a reduction in turn-around time per host team after making a few successful contributions to that host team.
+This is effect can be observed with Open Source as well, you can read more about it [here](https://doi.org/10.1109/MS.2013.95). 
```

Comment:
```suggestion
This is effect can be observed with Open Source as well, you can read more about it [here](https://doi.org/10.1109/MS.2013.95). 
Unfortunately, this paper access restricted. 
Sometimes your employer has a subscription enabling access, otherwise public university libraries often allow access for guests, too.
```

Reply:
Since many readers will not have access to this content can we move it to some sort of "references" section at the bottom instead of having it inline?

Reply:
Thank you!